src/main/scala/bloop/integrations/gradle/BloopPlugin.scala

```diff
@@ -0,0 +1,38 @@
+package bloop.integrations.gradle
+
+import bloop.integrations.gradle.syntax._
+import bloop.integrations.gradle.tasks.BloopInstallTask
+import bloop.integrations.gradle.tasks.ConfigureBloopInstallTask
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+/**
+ * Main entry point of the gradle bloop plugin.
+ *
+ * The bloop plugin defines two tasks:
+ *
+ *   1. `configureBloopInstall`: responsible to set up the environment and
+ *      force artifact resolution. 2. `bloopInstall`: responsible of generating
+ *      the bloop config files from the configured data.
```

Comment:
Something is wrong with the formatting here.

```suggestion
 * 1. `configureBloopInstall`: responsible to set up the environment and
 * force artifact resolution.
 * 2. `bloopInstall`: responsible of generating the bloop config files from
 * the configured data.
```

Reply:
Ah weird, fixed this.